etc/config/dart.amazon.properties

```diff
@@ -5,11 +5,19 @@ supportsExecute=true
 compilerType=dart
 objdumper=/opt/compiler-explorer/gcc-11.1.0/bin/objdump
 
-group.dart.compilers=dart2171:dart2161:dart2151:dart2144:dart2134:dart2124:dart2105:dart293:dart284
+group.dart.compilers=dev:dart2185:dart2171:dart2161:dart2151:dart2144:dart2134:dart2124:dart2105:dart293:dart284
```

Comment:
Can you please change `dev` to `dartdev`? Compiler ids are global and having one called `dev` is not ideal

Reply:
Thanks, done. I didn't realize they those were global ids. I guess in that case you may also want to rename the rust variants that are only called "nightly" and "beta".

Reply:
Oh nice catch! I didn't know those versions had that id.

Unfortunately IDs are inmutable so we can't change those once they go live, so we're stuck with those bad apples until the end of times, yay!